dev/jobs/prep.sh

```diff
@@ -78,16 +82,19 @@ fi
 
 if [[ ${PROCESS_TROPCY} = "YES" ]]; then
 
-    export COMINsyn=${COMINsyn:-$(compath.py gfs/prod/syndat)}
     export ARCHSYND=${ROTDIR}/syndat
-    if [[ ! -d ${ARCHSYND} ]]; then mkdir -p "${ARCHSYND}"; fi
+    if [[ ! -d ${ARCHSYND} ]]; then
+       mkdir -p "${ARCHSYND}"
+    fi
     if [[ ! -s ${ARCHSYND}/syndat_akavit ]]; then
         for file in syndat_akavit syndat_dateck syndat_stmcat.scr syndat_stmcat syndat_sthisto syndat_sthista ; do
-            cp "${COMINsyn}/${file}" "${ARCHSYND}"/.
+            cpreq "${COMIN_SYNDAT}/${file}" "${ARCHSYND}"/.
```

Comment:
I think `COMINsyndat` is the right name, given that there is a package named `syndat`.
We should update every reference to `COMINsyndat` instead of `COMINsyn` (except in the TC package where it for some reason uses `COMINsyn`).  I definitely would not creating `COMIN_SYNDAT`

Reply:
I was going to leave it as `COMINsyndat` as well but this is a GFS folder that lives at the top of GFS com (`../com/gfs/v16.3/syndat`) alongside the `RUN.PDY` folders, so it's internal to the GFS. I've been waffling still on changing it back...thoughts?

Reply:
Discussed offline and agreed to change back to `COMINsyndat`. Will do that now.